### ISSUE TITLE
[BUG] docker status Restarting (135)

### ISSUE THREAD

**Issue by jerryc127** [FreshRSS/FreshRSS#5460]
**Describe the bug**

freshrss/freshrss:edge-arm

After updating to the latest version today, the webpage is inaccessible, and when checking the status of Docker, it shows "Restarting (135)".
![image](https://github.com/FreshRSS/FreshRSS/assets/16351105/88347351-572f-4b6d-b8f7-c2930486dee7)

**To Reproduce**
1. Go to '…'
2. Click on '…'
3. Scroll down to '…'

**Expected behavior**

the docker can run 

**Screenshots**

**Environment information (please complete the following information):**
- OS: [e.g. Ubuntu 20.04]  Ubuntu 22.04.2 LTS
- Browser: [e.g. Firefox 86] 
- FreshRSS version: [e.g. 1.17.1]  freshrss/freshrss:edge-arm
- Database version: [e.g. Mysql 5.7] SQLlite
- PHP version: [e.g. PHP 7.4]
- Installation type: [e.g. Yunohost] self-hosted with docker compose

**Additional context**

```
version: "3"
services:
  freshrss:
    image: freshrss/freshrss:edge-arm
    container_name: freshrss
    hostname: freshrss
    restart: unless-stopped
    ports:
      - 8080:80
    logging:
      options:
        max-size: 10m
    volumes:
      - ./data:/var/www/FreshRSS/data
      - ./extensions:/var/www/FreshRSS/extensions
    environment:
      TZ: Asia/Hong_Kong
      CRON_MIN: '*/10'

volumes:
  data:
  extensions:
```

**Comment by Alkarex** [comment]
Hello,
Could you please provide some Docker logs so we can see what errors you get?

**Comment by jerryc127** [comment]
how to find the log

I try to docker-compose logs, nothing output

i find some log in /var/log/syslog

```
Jun 13 16:23:06 myw kernel: [1714259.786430] br-3b395b192faa: port 1(vethc6e4b5a) entered blocking state
Jun 13 16:23:06 myw kernel: [1714259.786436] br-3b395b192faa: port 1(vethc6e4b5a) entered disabled state
Jun 13 16:23:06 myw kernel: [1714259.787623] device vethc6e4b5a entered promiscuous mode
Jun 13 16:23:06 myw systemd-networkd[3317251]: vethc6e4b5a: Link UP
Jun 13 16:23:06 myw dockerd[856725]: time="2023-06-13T16:23:06.719967783+08:00" level=info msg="No non-localhost DNS nameservers are left in resolv.conf. Using default external servers: [nameserver 8.8.8.8 nameserver 8.8.4.4]"
Jun 13 16:23:06 myw dockerd[856725]: time="2023-06-13T16:23:06.720001543+08:00" level=info msg="IPv6 enabled; Adding default IPv6 external servers: [nameserver 2001:4860:4860::8888 nameserver 2001:4860:4860::8844]"
Jun 13 16:23:06 myw networkd-dispatcher[950]: WARNING:Unknown index 5009 seen, reloading interface list
Jun 13 16:23:06 myw containerd[973]: time="2023-06-13T16:23:06.767086588+08:00" level=info msg="loading plugin \"io.containerd.event.v1.publisher\"..." runtime=io.containerd.runc.v2 type=io.containerd.event.v1
Jun 13 16:23:06 myw containerd[973]: time="2023-06-13T16:23:06.767154309+08:00" level=info msg="loading plugin \"io.containerd.internal.v1.shutdown\"..." runtime=io.containerd.runc.v2 type=io.containerd.internal.v1
Jun 13 16:23:06 myw containerd[973]: time="2023-06-13T16:23:06.767164509+08:00" level=info msg="loading plugin \"io.containerd.ttrpc.v1.task\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
Jun 13 16:23:06 myw containerd[973]: time="2023-06-13T16:23:06.767817672+08:00" level=info msg="starting signal loop" namespace=moby path=/run/containerd/io.containerd.runtime.v2.task/moby/31e291784d87a1dc2ada8133075e9366786a6bb83e084c4dd91ac3128c110048 pid=118879 runtime=io.containerd.runc.v2
Jun 13 16:23:06 myw systemd[1]: run-docker-runtime\x2drunc-moby-31e291784d87a1dc2ada8133075e9366786a6bb83e084c4dd91ac3128c110048-runc.JZ9URz.mount: Deactivated successfully.
Jun 13 16:23:06 myw systemd[1]: Started libcontainer container 31e291784d87a1dc2ada8133075e9366786a6bb83e084c4dd91ac3128c110048.
Jun 13 16:23:06 myw kernel: [1714259.982649] eth0: renamed from veth01c75cd
Jun 13 16:23:06 myw systemd-networkd[3317251]: vethc6e4b5a: Gained carrier
Jun 13 16:23:06 myw systemd-networkd[3317251]: br-3b395b192faa: Gained carrier
Jun 13 16:23:06 myw kernel: [1714259.997536] IPv6: ADDRCONF(NETDEV_CHANGE): vethc6e4b5a: link becomes ready
Jun 13 16:23:06 myw kernel: [1714259.997608] br-3b395b192faa: port 1(vethc6e4b5a) entered blocking state
Jun 13 16:23:06 myw kernel: [1714259.997611] br-3b395b192faa: port 1(vethc6e4b5a) entered forwarding state
Jun 13 16:23:07 myw systemd[1]: docker-31e291784d87a1dc2ada8133075e9366786a6bb83e084c4dd91ac3128c110048.scope: Deactivated successfully.
Jun 13 16:23:07 myw dockerd[856725]: time="2023-06-13T16:23:07.259220174+08:00" level=info msg="ignoring event" container=31e291784d87a1dc2ada8133075e9366786a6bb83e084c4dd91ac3128c110048 module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
Jun 13 16:23:07 myw containerd[973]: time="2023-06-13T16:23:07.259346895+08:00" level=info msg="shim disconnected" id=31e291784d87a1dc2ada8133075e9366786a6bb83e084c4dd91ac3128c110048
Jun 13 16:23:07 myw containerd[973]: time="2023-06-13T16:23:07.259392055+08:00" level=warning msg="cleaning up after shim disconnected" id=31e291784d87a1dc2ada8133075e9366786a6bb83e084c4dd91ac3128c110048 namespace=moby
Jun 13 16:23:07 myw containerd[973]: time="2023-06-13T16:23:07.259522496+08:00" level=info msg="cleaning up dead shim"
Jun 13 16:23:07 myw containerd[973]: time="2023-06-13T16:23:07.268738536+08:00" level=warning msg="cleanup warnings time=\"2023-06-13T16:23:07+08:00\" level=info msg=\"starting signal loop\" namespace=moby pid=118972 runtime=io.containerd.runc.v2\n"
Jun 13 16:23:07 myw systemd-networkd[3317251]: vethc6e4b5a: Lost carrier
Jun 13 16:23:07 myw kernel: [1714260.407891] br-3b395b192faa: port 1(vethc6e4b5a) entered disabled state
Jun 13 16:23:07 myw kernel: [1714260.407950] veth01c75cd: renamed from eth0
Jun 13 16:23:07 myw networkd-dispatcher[950]: WARNING:Unknown index 5009 seen, reloading interface list
Jun 13 16:23:07 myw systemd-networkd[3317251]: vethc6e4b5a: Link DOWN
Jun 13 16:23:07 myw kernel: [1714260.461432] br-3b395b192faa: port 1(vethc6e4b5a) entered disabled state
Jun 13 16:23:07 myw kernel: [1714260.463846] device vethc6e4b5a left promiscuous mode
Jun 13 16:23:07 myw kernel: [1714260.463850] br-3b395b192faa: port 1(vethc6e4b5a) entered disabled state
Jun 13 16:23:07 myw networkd-dispatcher[950]: ERROR:Unknown interface index 5009 seen even after reload
Jun 13 16:23:07 myw networkd-dispatcher[950]: WARNING:Unknown index 5009 seen, reloading interface list
Jun 13 16:23:07 myw networkd-dispatcher[950]: ERROR:Unknown interface index 5009 seen even after reload
Jun 13 16:23:07 myw systemd[1]: run-docker-netns-5bb2d9244635.mount: Deactivated successfully.
Jun 13 16:23:07 myw systemd[1]: var-lib-docker-overlay2-121f449394f357038cd62d5fea9d19809ef29639b851f055a9a43a679494ac96-merged.mount: Deactivated successfully.
Jun 13 16:23:07 myw systemd-networkd[3317251]: br-3b395b192faa: Lost carrier
```

**Comment by Alkarex** [comment]
Could you try a couple of things, to get more info?

1) Start the docker container manually:

```sh
sudo docker run --rm -p 8080:80 -e TZ=Asia/Hong_Kong -e 'CRON_MIN=*/10' -v ${PWD}/data:/var/www/FreshRSS/data -v ${PWD}/extensios:/var/www/FreshRSS/extensions --name freshrss freshrss/freshrss:edge-arm
```

2) Use a local build https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#docker-compose

And please do so with both `Docker/Dockerfile-Alpine` and `Docker/Dockerfile` to see whether the bug might be related to our Debian image

```yaml
version: "3"
services:
  freshrss:
    image: freshrss/freshrss:edge-custom
    # Optional build section if you want to build the image locally:
    build:
      # Pick #latest (stable release) or #edge (rolling release) or a specific release like #1.21.0
      context: https://github.com/FreshRSS/FreshRSS.git#edge
      dockerfile: Docker/Dockerfile-Alpine
    container_name: freshrss
    hostname: freshrss
    restart: unless-stopped
    ports:
      - 8080:80
    logging:
      options:
        max-size: 10m
    volumes:
      - ./data:/var/www/FreshRSS/data
      - ./extensions:/var/www/FreshRSS/extensions
    environment:
      TZ: Asia/Hong_Kong
      CRON_MIN: '*/10'

volumes:
  data:
  extensions:
```

**Comment by Alkarex** [comment]
I found an ARM board to test and I can reproduce the bug

**Comment by jerryc127** [comment]
1. i run this command, but i check `docker ps`, cannot found the freshrss

2. both Docker/Dockerfile-Alpine and Docker/Dockerfile, i use `docker compose build freshrss` to create and run , it work.

**Comment by jerryc127** [comment]
> I found an ARM board to test and I can reproduce the bug

OK

**Comment by Alkarex** [comment]
Seems to be due to https://github.com/FreshRSS/FreshRSS/pull/5351

**Comment by Alkarex** [comment]
Fixed in https://github.com/FreshRSS/FreshRSS/pull/5463
I have merged and new images have been built on Docker Hub. Could you please try again?

**Comment by Alkarex** [comment]
Upstream bug / discussion: https://github.com/OpenIDC/mod_auth_openidc/discussions/1067

**Comment by jerryc127** [comment]
> Fixed in #5463 I have merged and new images have been built on Docker Hub. Could you please try again?

It work now.
Excellent!
